test/unit/template_exporter_test.rb

```diff
@@ -67,10 +101,10 @@ class TemplateExporterTest < ActiveSupport::TestCase
         assert_equal "/tmp/provisioning_templates/snippet", @exporter.get_dump_dir(@template)
       end
 
-      test 'does not appends anything for template without kind' do
+      test 'appends _templates for ptables' do
         @exporter.instance_variable_set('@dir', '/tmp')
         template = FactoryBot.create(:ptable)
-        assert_equal "/tmp/ptables/", @exporter.get_dump_dir(template)
+        assert_equal "/tmp/partition_tables_templates/", @exporter.get_dump_dir(template)
```

Comment:
this does not seem to work, export dir is still ptables, it would also become incompatible with community-templates repo, is the change intentional? I think it's ready for merge after reverting this